crates/typst-library/src/pdf/accessibility.rs

```diff
@@ -36,7 +37,17 @@ pub enum ArtifactKind {
     Other,
 }
 
-// TODO: feature gate
+/// Add a summary of the table's purpose and structure.
+/// This will be available for assistive techonologies (such as screen readers).
```

Comment:
The first line and the rest should be separated by a blank line. The first line used as a "oneliner" in various places.

The docs should also mention that this is feature gated, similarly to how the `target` function does it. (The individual html function don't because the whole module is gated.)

A short example for how to use this would be nice, too.

Same applies to the other two functions.